projects/1.18/assets/epic-fight-mod/epicfight/lang/zh_cn.json

```
@@ -1,174 +1,272 @@
+	"epicfight.weapon_category.axe": "斧",
+	"epicfight.weapon_category.fist": "拳",
+	"epicfight.weapon_category.greatsword": "大剑",
+	"epicfight.weapon_category.hoe": "锄",
+	"epicfight.weapon_category.pickaxe": "镐",
+	"epicfight.weapon_category.shovel": "铲",
+	"epicfight.weapon_category.sword": "剑",
+	"epicfight.weapon_category.uchigatana": "打刀",
+	"epicfight.weapon_category.spear": "长矛",
+	"epicfight.weapon_category.tachi": "太刀",
+	"epicfight.weapon_category.trident": "三叉戟",
+	"epicfight.weapon_category.longsword": "长剑",
+	"epicfight.weapon_category.dagger": "匕首",
+	"epicfight.weapon_category.shield": "盾",
+	"epicfight.weapon_category.ranged": "远程",
+
+	"epicfight.skill_category.dodge": "闪避",
+	"epicfight.skill_category.passive": "被动",
+	"epicfight.skill_category.guard": "防御",
+	
+	"epicfight.skill_slot.dodge": "闪避",
+	"epicfight.skill_slot.passive1": "被动 1 ",
+	"epicfight.skill_slot.passive2": "被动 2",
+	"epicfight.skill_slot.passive3": "被动 3",
+	"epicfight.skill_slot.guard": "防御",
+	"epicfight.skill_slot.identity": "属性",
+	"epicfight.skill_slot.mover": "行动",
+
+	"gameMode.epicfight.mining": "挖掘",
+	"gameMode.epicfight.battle": "战斗",
+
+	"gamerule.doVanillaAttack": "启用原版攻击",
+	"gamerule.hasFallAnimation": "启用落地动画",
+	"gamerule.keepSkills": "死后保留技能",
+	"gamerule.weightPenalty": "重量惩罚适用于攻击速度。",
+	"gamerule.disableEntityUI": "关闭所有玩家的生命条和目标指示器",
+	"gamerule.globalStun": "启用全局眩晕",
+	"gamerule.canSwitchCombat": "启用切换战斗/挖掘模式",
+
+	"gui.epicfight.skill_edit": "技能",
+	"gui.epicfight.filter_animation.on": "滤镜动画：开",
+	"gui.epicfight.filter_animation.off": "滤镜动画：关",
+
+	"gui.epicfight.health_bar_show_option.none": "生命条显示选项：无",
+	"gui.epicfight.health_bar_show_option.hurt": "生命条显示选项：受伤",
+	"gui.epicfight.health_bar_show_option.target": "生命条显示选项：目标",
+	"gui.epicfight.target_indicator.on": "显示目标指示器：开",
+	"gui.epicfight.target_indicator.off": "显示目标指示器：关",
+
+	"gui.epicfight.ui_setup": "游戏内UI调整",
+	"gui.epicfight.ui_setup.tooltip": "打开UI设置画面",
+	"gui.epicfight.aim_helper.on": "显示瞄准辅助线：开",
+	"gui.epicfight.aim_helper.off": "显示瞄准辅助线：关",
+	"gui.epicfight.aim_helper_color": "瞄准辅助线颜色",
+	"gui.epicfight.long_press_counter": "长按时间：%s",
+	"gui.epicfight.camera_auto_switch.on": "自动切换视角：开",
+	"gui.epicfight.camera_auto_switch.off": "自动切换视角：关",
+	"gui.epicfight.auto_preparation.on": "自动备战：开",
+	"gui.epicfight.auto_preparation.off": "自动备战：关",
+	"gui.epicfight.off_blood_effects.on": "血液效果：开",
+	"gui.epicfight.off_blood_effects.off": "血液效果：关",
+	"gui.epicfight.no_mining_in_combat.on": "战斗中禁止挖掘：开",
+	"gui.epicfight.no_mining_in_combat.off": "战斗中禁止挖掘：关",
+	"gui.epicfight.auto_switching_items": "自动切换物品",
+	"gui.epicfight.export_custom_armor": "导出盔甲模型",
+	"gui.epicfight.filter_animation.tooltip": "在挖掘模式下打开/关闭动画模型",
+	"gui.epicfight.health_bar_show_option.tooltip": "哪些实体应该显示生命条？\n\n受伤：生命值不满的实体。\n目标：玩家当前瞄准的目标。\n无：禁用生命条。",
+	"gui.epicfight.target_indicator.tooltip": "打开/关闭目标指示器",
+	"gui.epicfight.long_press_counter.tooltip": "判定长按所需的多少刻，用于武器的特殊攻击和闪避",

```

```suggestion
	"gui.epicfight.long_press_counter.tooltip": "判定为长按所需的刻，用于武器的特殊攻击和闪避",

```
主要修改意见在于how many在陈述句中译作x的数量
ticks不译？需要其他审阅人员帮助。